### ISSUE TITLE
Add Custom query support to MSSQL Integration

### ISSUE THREAD

**Issue by jammiemil** [grafana/agent#3799]
The upstream sql_exporter  supports defining custom metrics/queries to run against the SQL Server, is it possible to extend this functionality to Grafana Agent?

https://github.com/burningalchemist/sql_exporter#collectors

Supporting the in-line functionality might fit the grafana agent config better than having to create additional files

**Comment by prajul-nambiar** [comment]
Yes, i agree. This will be good feature to have and it would make grafana agent more powerful.

Is this on cards, Grafana team?
